src/test/java/seedu/address/model/client/TimezoneTest.java

```diff
@@ -32,65 +32,106 @@ public void isValidTimezone() {
         // invalid timezone
         assertFalse(Timezone.isValidTimezone("")); // empty string
         assertFalse(Timezone.isValidTimezone(" ")); // spaces only
-        assertFalse(Timezone.isValidTimezone("UTC+8")); // does not start with GMT
-        assertFalse(Timezone.isValidTimezone("GM+8")); // misspelled GMT
-        assertFalse(Timezone.isValidTimezone("GMT+15")); // out of range (positive)
-        assertFalse(Timezone.isValidTimezone("GMT-13")); // out of range (negative)
+        assertFalse(Timezone.isValidTimezone("UTC+00:00")); // does not start with GMT
+        assertFalse(Timezone.isValidTimezone("GM+00:00")); // misspelled GMT
+        assertFalse(Timezone.isValidTimezone("GMT+15:00")); // out of range (positive)
+        assertFalse(Timezone.isValidTimezone("GMT-13:00")); // out of range (negative)
+        assertFalse(Timezone.isValidTimezone("GMT+13:30")); // unrecognised timezone
         assertFalse(Timezone.isValidTimezone("GMT+654657987654456")); // large number
```

Comment:
Should add a few more negative test cases like "GMT" and "GMT+" and "GMT-", and maybe "gmt+08:00", and "GMT+8" and "GMT+08", and "GMT+8:00", and "GMT+08:0"